scripts/scripts.js

```diff
@@ -8,13 +7,17 @@ import {
   decorateSections,
   decorateBlocks,
   decorateTemplateAndTheme,
-  waitForLCP,
+  init,
   loadBlocks,
   loadCSS,
+  waitForLCP,
+  withPlugin,
 } from './lib-franklin.js';
 
 const LCP_BLOCKS = []; // add your LCP blocks to the list
-window.hlx.RUM_GENERATION = 'project-1'; // add your RUM generation information here
+
+const rum = await withPlugin('./plugins/rum.js');
```

Comment:
```suggestion
const rum = await withPlugin('./plugins/rum.js', 'project-1');
```
to make clear that you have to touch here and not in `rum.js`.